packages/lesswrong/components/users/ViewSubscriptionsPage.tsx

```diff
@@ -1,272 +1,193 @@
-import React, {ReactNode} from 'react';
```

Comment:
Most of the changes here are refactoring. The only actual additions are wrapping the page in `CountItemsContextProvider` and the `NoSubscriptionsMessage` component which is shown when there are no subscriptions.